packages/design/buildIconStyles.js

```diff
@@ -27,6 +28,18 @@ postcss([
       "/* This file is automatically generated by buildIconStyle.js and should not be edited. */\n" +
       "export const iconStyles = " +
       JSON.stringify(calculated, undefined, 2);
+    try {
+      fs.readdirSync("./node_modules", (err, files) => {
+        if (err) {
+          console.log("error", err);
+        }
+        files.forEach(file => {
+          console.log(file);
+        });
+      });
+    } catch (e) {
+      console.log("error", e);
+    }
```

Comment:
Do we need to log everything here?

Reply:
I kept this from the previous version however I do not think we need to do the logging in the callback. I think logging the error is still valuable for debugging failures.